src/Esfa.Vacancy.UnitTests/CreateApprenticeship/Application/GivenACreateApprenticeshipRequestValidator/WhenValidatingApplicationMethod/AndApplicationMethodIsOffline.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Esfa.Vacancy.Application.Commands.CreateApprenticeship;
+using Esfa.Vacancy.Application.Commands.CreateApprenticeship.Validators;
+using Esfa.Vacancy.Domain.Validation;
+using FluentValidation.TestHelper;
+using NUnit.Framework;
+
+namespace Esfa.Vacancy.UnitTests.CreateApprenticeship.Application.GivenACreateApprenticeshipRequestValidator.
+    WhenValidatingApplicationMethod
+{
```

Comment:
This test will be redundant after changing the enum to not include this option

Reply:
Removed along with NotSpecified test